webpack.config.js

```diff
@@ -26,6 +26,12 @@ const plugins = [
             to: staticPath
         }
     ]),
+    new CopyWebpackPlugin([
```

Comment:
I'll address this ahead of time:
This was required to get the wasm file served statically, as that seems the only way.
It seems to be impossible to pack the wasm resources with webpack.

Reply:
if nothing else, there is an example relating to wasm in the webpack repos: https://github.com/webpack/webpack/blob/master/examples/wasm-simple/webpack.config.js

the problem with the CopyWebackPlugin is that it doesn't put a hash in the resulting filename, which causes us problems if/when the file changes.

Our CDN is meant to be immutable, so we really need hashes in all filenames.

(the es6-shim above your addition is my fault and I take all responsibility for it).

I am a bit unclear what is actually referencing/loading `graphvizlib.wasm` to begin with - is there some light you can shed there?

Reply:
Sure thing.

I think currently the whole mechanism in d3-graphviz / hpcc-wasm for loading this is... hacky.

I tried to get some clarity as to what is supposed to happen here:
https://github.com/hpcc-systems/hpcc-js-wasm/issues/26

The issue I was trying to resolve is that without loading the wasm file, graphvizlib is DOA, naturally, this was failing for me:

https://github.com/hpcc-systems/hpcc-js-wasm/blob/1e4f67e26a94f9de6f6b1a7b9ceed85bceed3007/src/util.ts#L25

I admit this is a bit over my head in terms of what webpack does and how it interacts / intercepts calls to load `.wasm` files.